src/index.html

```diff
@@ -12,7 +12,43 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="container">
+      <section class="calendar calendar--month-length-31">
+        <div
+          class="calendar__day calendar__day--1 calendar__day--start-day-sun"
```

Comment:
```suggestion
          class="calendar__day"
```
move this logic to the section tag